src/main/java/week2/Node.java

```diff
@@ -0,0 +1,39 @@
+package week2;
```

Comment:
would this be possible given that the `Node` class is used in the Test Class to construct the tree